src/plugins/interactive_setup/public/app.scss

```diff
@@ -0,0 +1,21 @@
+.interactiveSetup {
+  @include kibanaFullScreenGraphics;
+}
+
+.interactiveSetup__header {
+  padding: $euiSizeXL;
+}
+
+.interactiveSetup__logo {
+  @include kibanaCircleLogo;
+  @include euiBottomShadowMedium;
+
+  margin-bottom: $euiSizeXL;
+}
+
+.interactiveSetup__content {
+  margin: auto;
+  max-width: map-get($euiBreakpoints, 's') - $euiSizeXL;
+  padding-left: $euiSizeXL;
+  padding-right: $euiSizeXL;
+}
```

Comment:
The background is acting up at certain resolutions

![image](https://user-images.githubusercontent.com/4016496/129841203-8ccbcae7-03a9-4fee-aba3-8dbcc459c6c7.png)

You can fix it this by adding:


```suggestion
  z-index: 10;
  position: relative;
 }
```

Reply:
![image](https://user-images.githubusercontent.com/4016496/129844724-b3d9c8a4-7cd8-44e8-a3c1-6e70310a3862.png) 

We need to add some padding at the bottom here. To replicate see window at 1024 x 768 (iPad).

Reply:
Also, I think it might be useful to use two different classes for the header and the content like in the login page (`loginWelcome__header` and `loginWelcome__content`).

Reply:
> The background is acting up at certain resolutions
> 
> You can fix it this by adding:

Ah, thanks, that's what that was for :)